src/pages/projectSettings/tabs/GithubCommitQueueTab/GithubCommitQueueTab.tsx

```diff
@@ -51,7 +49,7 @@ export const GithubCommitQueueTab: React.VFC<TabProps> = ({
     GithubProjectConflictsQueryVariables
   >(GET_GITHUB_PROJECT_CONFLICTS, {
     skip: projectType === ProjectType.Repo,
-    variables: { projectId: identifier },
```

Comment:
All other project settings related queries were using the `projectID` so I changed this query to use `projectID` as well. (Although it really doesn't matter because of how [`byID`](https://github.com/evergreen-ci/evergreen/blob/d1f28a88613c3789e22ad4b1ddd5f2c3935b284c/model/project_ref.go#L1380-L1387) is defined)

Reply:
Sorry if I missed this but why does this use `projectId` instead of `identifier`?

Reply:
Project settings uses `projectId` for a lot of different mutations. I assume that part of the reason is that repos don't have a concept of an `identifier`.

<img width="386" alt="Screen Shot 2023-01-31 at 7 06 23 PM" src="https://user-images.githubusercontent.com/47064971/216086384-d91831e6-608e-426c-9cec-a999060f0673.png">

This is fine and correct because we actually do send the `ID` to these queries on Spruce.

https://github.com/evergreen-ci/spruce/blob/19d98d6775831ce1ec8d4f681e1f7440839b7e69/src/pages/projectSettings/Tabs.tsx#L42

So all I'm doing here is using the `projectID` as the query specifies. There could be a more thorough audit of project settings queries/mutations but it would need a larger effort.